### PR TITLE
Update lutris.appdata.xml

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

File: share/appdata/lutris.appdata.xml [hunk 1]
```diff
@@ -1,35 +1,57 @@
...
-  <summary>Open gaming platform for Linux</summary>
-  <description>
-    <p>
-        Lutris is an open gaming platform for Linux. It helps you install and
-        manage your games in a unified interface.  Our goal is to support every
-        game which runs on Linux, from native to Windows games (via Wine) to
-        emulators and browser games.
-    </p>
-  </description>
+  <summary>Lutris is an open source gaming platform for Linux distributions</summary>
```
[the remark this answers is not in the document]
Comment (AlexanderRavenheart) [reply share/appdata/lutris.appdata.xml]: It is the proper term, as Linux is actually just the kernel of the OS. If you want to have the full proper name it should actually be GNU/Linux as described [here](https://en.wikipedia.org/wiki/Linux) but I don't think that's necessary.

File: share/appdata/lutris.appdata.xml [hunk 2]
```diff
@@ -1,35 +1,56 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component type="desktop">
```
Comment (TingPing) [inline share/appdata/lutris.appdata.xml]: You are downgrading the format...

Re-run `appstream-util upgrade` on the file...

File: share/appdata/lutris.appdata.xml [hunk 3]
```diff
@@ -1,35 +1,37 @@
...
+  <description xml:lang="pt_BR"><p>Lutris é uma plataforma de jogos aberta para distribuições Linux. Ele ajuda você a instalar e gerenciar seus jogos em uma interface unificada. Nosso objetivo é suportar todos os jogos que rodam no Linux, desde jogos nativos até Windows (via Wine), emuladores e jogos de navegador. O aplicativo de desktop e o site são softwares livres, suas contribuições são bem vindas!</p></description>
+  <description xml:lang="es">Lutris es una plataforma de juegos abierta para distribuciones Linux. Te ayuda a instalar y administrar tus juegos en una interfaz unificada. Nuestro objetivo es apoyar todos los juegos que se ejecutan en Linux, desde juegos nativos a juegos de Windows (a través de Wine) a emuladores y juegos de navegador. La aplicación de escritorio y el sitio web son software libre, ¡sus contribuciones son bienvenidas!<p></p></description>
+  <launchable type="desktop-id">lutris.desktop</launchable>
+  <project_license>GPL-3.0</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <releases>
+    <release version="0.4.19" date="2018-08-05">
```
[the remark this answers is not in the document]
Comment (TingPing) [reply share/appdata/lutris.appdata.xml]: You actually really should maintain them here, these are shown in all graphical stores these days from GNOME-Software, Plasma Discover, Pop Shop/Elementary's store, etc.

I suggest a script that automatically converts from one format to the other.

File: share/appdata/lutris.appdata.xml [hunk 4]
```diff
@@ -1,35 +1,28 @@
...
-  <summary>Open gaming platform for Linux</summary>
-  <description>
-    <p>
-        Lutris is an open gaming platform for Linux. It helps you install and
-        manage your games in a unified interface.  Our goal is to support every
-        game which runs on Linux, from native to Windows games (via Wine) to
-        emulators and browser games.
-    </p>
-  </description>
+  <summary>Lutris is an open source gaming platform for Linux distribution</summary>
```
Comment (strycore) [inline share/appdata/lutris.appdata.xml]: this is now grammatically wrong

**Comment by strycore** [comment]
looks good

**Comment by strycore** [comment]
although, next time, please rebase and squash your commits if you send a bunch with the same commit message.

**Pull request merged**